### ISSUE TITLE
dummy_loss 

### ISSUE THREAD

**Issue by yanyantech** [abdxy/style-transfer-keras#1]
I'm interested in your project. I ran your project on the GPU for 12 hours without any results. I only used 100 pictures.
 I think there may be some problems with your project. 
For example, vgg1. compile (optimizer, loss = dummy_loss), which I don't understand. 
There are no details about how to generate pictures in you code.  
Can you my questions or contact me  shiyan@yanyantech.com
